docs/dev/hacking-guide.md

```diff
@@ -0,0 +1,193 @@
+# Cluster cloud controller manage operator - Hacking guide
+
+## How to run a component locally for testing
+
+Prerequisites:
+
+```bash
+git checkout github.com/openshit/$repository_name
+cd $repository_name
+```
+
+Make sure your `$KUBECONFIG` is set properly, because it will be used to interact with your cluster.
+
+First step is to scale down the cluster version operator.
+
+> “At the heart of OpenShift is an operator called the Cluster Version Operator. This operator watches the deployments and images related to the core OpenShift services, and will prevent a user from changing these details. If I want to replace the core OpenShift services I will need to scale this operator down.”
+
+```bash
+oc scale --replicas=0 deployment/cluster-version-operator -n openshift-cluster-version
+```
+
+Second step is scaling down the CCCMO. The operator watches for deployed resources to be running and will prevent them from scaling down.
+
+```bash
+oc scale deployment/cluster-cloud-controller-manager -n openshift-cloud-controller-manager-operator --replicas=0
+```
+
+Finally, once all has been scaled down you can compile and run the controller.
+
+You can copy or edit `hack/example-images.json` with the provider image you want to run.
```

Comment:
i think you should combine these two sentences, it will make more sense. 
```suggestion
Finally, once all has been scaled down you can compile and run the controller. If you need to run additional images you can copy or edit `hack/example-images.json` to add the images you want to run.
```

Reply:
Amended it slightly, so this sentence makes more sense.